etabotsite/etabotapp/urls.py

```diff
@@ -43,7 +50,7 @@
     url(r'^account/password_reset/complete/$',
         auth_views.PasswordResetCompleteView.as_view(),
         name='password_reset_complete'),
-
+    url(r'^jira_callback', jira_callback, name='jira_callback'),
```

Comment:
can we rename it to something more specific? `jira_callback` seems to generic to me and by reading the code, I don't quite get what it does other than simply return a page saying `OK`.

Reply:
that is usually how generic callback urls are. The complexity is in the passed parameters. There is one callback per external API.